examples/model/alcap/run.jl

```diff
@@ -9,7 +9,7 @@ using CarboKitten.Components
 using CarboKitten.Components.Common
 using CarboKitten.Model: ALCAP2 as ALCAP
 using CarboKitten.Export: data_export, CSV
-
+using CarboKitten.BoundaryTrait
```

Comment:
BoundaryTrait should already be imported from Components.Common

Reply:
I'll make a new branch to store the solution

Reply:
Yes it should, it runs yesterday without having this line. But today it keeps asking me what is Coast. 
is it because in Common.jl: `export Box, axes, Boundary, Shelf, Periodic, Reflected, TimeProperties`, we have not export the 'Coast', although we set `const Shelf = Coast` in BoundaryTrait?

Reply:
Yes, that's right. Fixed now in branch 83-urgent-fixes #83